tests/unit/sproxyd.js

```diff
@@ -92,10 +103,11 @@ function handler(req, res) {
     }
 }
 
+const clientCustomPath =
+    new Sproxy({ bootstrap: ['127.0.0.1:9001:/proxy/arc/'] });
```

Comment:
This does seem to differ from the intended use of the parsing method (https://github.com/scality/sproxydclient/blob/master/lib/sproxyd.js#L39). But I trust this is the best approach here.